_site.yml

```diff
@@ -1,19 +1,8 @@
 name: "WRDS_R_intro"
-output_dir: docs
-output:
-  bookdown::html_document2:
-    theme: cosmo
-    number_sections: false
-    highlight: textmate
-    toc: true
-    toc_float:
-        collapsed: true
-    css: styles.css
-    lib_dir: site_libs
-    self_contained: no
-    include:
-        in_header: assets/header.html
-        after_body: assets/footer.html
+title: "R for Water Resources Data Science"
+description: |
+  R for Water Resources Data Science
+output_dir: "_site"
```

Comment:
let's add:
`highlight: rstudio` 
here as I think that is a nice default code/formatting option that will match what folks see when they open RStudio. Or at least a place holder to use for whatever our final code format will be.

Reply:
good idea to match what the users see on the site and in RStudio. Updated and I'll merge now.